src/CommentsColumn/Comment/Comment.css

```diff
@@ -7,6 +7,9 @@ button {
   display: flex;
   flex-direction: column;
   position: relative;
+  border-width: 0.1px 0px;
```

Comment:
This can be done on one line, since only the bottom border needs to be styled
`border-bottom: 0.5px solid #c7c7c7`